### DISCUSSION TITLE
How to specify the collison of subbodies of a compound body?

### DISCUSSION THREAD

**Discussion by samedbayrakoglu** [enable3d/enable3d#159] [Help / Questions]
Hi,

let me explain the question with my simple case. I have a knife model, and it will consist of two bodies: Blade and Haft. They should move and rotate together. Therefore, I created a compound body (blade and haft) as a group but I wanna find out which subbody(blade or haft) collided with other bodies. How can I know it? Or do you have an alternative method suggestion?

thanks in advance..

**Comment by ThorstenBux** [comment]
@yandeu , I would highly appreciate any guidance on the above question. Thank you

**Comment by yandeu** [comment]
Unfortunately, I don't know. I don't even know of this is possible.

But, you could add a ghost body around your blade.
